test/functionalTest/cases/1039_ngsiv2_date_support/dates_in_uri_param_ranges.test

```diff
@@ -0,0 +1,192 @@
+echo
+echo
+
+
+echo "05. GET entities with birthdate FROM 2016-02-20T10:00:00.00Z TO 2016-02-26T10:00:00.00Z, see all three"
+echo "======================================================================================================"
+orionCurl --url '/v2/entities?q=birthdate==2016-02-20T10:00:00.00Z..2016-02-26T10:00:00.00Z&options=keyValues' --json
+echo
+echo
+
+
+echo "06. GET entities with birthdate FROM 2016-02-23T10:00:00.00Z TO 2016-02-24T10:00:00.00Z, see E23 and E24"
+echo "========================================================================================================"
+orionCurl --url '/v2/entities?q=birthdate==2016-02-23T10:00:00.00Z..2016-02-24T10:00:00.00Z&options=keyValues' --json
+echo
+echo
+
+
+--REGEXPECT--
+01. Create entity E24 with birthdate 2016-02-24T10:00:00.00Z
+============================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/E24
+Date: REGEX(.*)
+
+
+
+02. Create entity E23 with birthdate 2016-02-23T10:00:00.00Z
+============================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/E23
+Date: REGEX(.*)
+
+
+
+03. Create entity E22 with birthdate 2016-02-22T10:00:00.00Z
+============================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/E22
+Date: REGEX(.*)
+
+
+
+04. GET entities with birthdate FROM 2016-02-25T10:00:00.00Z TO 2016-02-26T10:00:00.00Z, see nothing
+====================================================================================================
+HTTP/1.1 200 OK
+Content-Length: 2
+Content-Type: application/json
+Date: REGEX(.*)
+
+[]
+
+
+05. GET entities with birthdate FROM 2016-02-20T10:00:00.00Z TO 2016-02-26T10:00:00.00Z, see all three
+======================================================================================================
+HTTP/1.1 200 OK
+Content-Length: 190
+Content-Type: application/json
+Date: REGEX(.*)
+
+[
+    {
+        "birthdate": "2016-02-24T10:00:00.00Z",
+        "id": "E24",
+        "type": null
+    },
+    {
+        "birthdate": "2016-02-23T10:00:00.00Z",
+        "id": "E23",
+        "type": null
+    },
+    {
+        "birthdate": "2016-02-22T10:00:00.00Z",
+        "id": "E22",
+        "type": null
+    }
+]
+
+
+06. GET entities with birthdate FROM 2016-02-23T10:00:00.00Z TO 2016-02-24T10:00:00.00Z, see E23 and E24
+========================================================================================================
+HTTP/1.1 200 OK
+Content-Length: 127
+Content-Type: application/json
+Date: REGEX(.*)
+
+[
+    {
+        "birthdate": "2016-02-24T10:00:00.00Z",
+        "id": "E24",
+        "type": null
+    },
+    {
+        "birthdate": "2016-02-23T10:00:00.00Z",
+        "id": "E23",
+        "type": null
+    }
+]
+
+
+--TEARDOWN--
+brokerStop CB
+dbDrop CB
```

Comment:
I mean something like this:

birthdate==2016-02-24T10:00:00.00Z,2016-02-25T10:00:00.00Z

We could leave it for latter, but I think the moment is now, when we have our mind "hot" with this topic. Remember how hard could be the "context change" after some weeks ;)